src/utils/paths.cpp

```diff
@@ -68,6 +70,23 @@ std::filesystem::path Paths::getUserLockFilePath() {
 }
 
 std::filesystem::path Paths::getSystemConfigFilePath() {
+  char *xdgConfigDirs = getenv("XDG_CONFIG_DIRS");
+
+  if (xdgConfigDirs != nullptr) {
+    char *rest = nullptr;
+    char *configDir = strtok_r(xdgConfigDirs, ":", &rest);
+
+    while (configDir != nullptr) {
+      std::filesystem::path configPath = std::filesystem::path{configDir};
+      configPath =
+          std::filesystem::path{configPath / "touchegg" / "touchegg.conf"};
+
+      if (std::filesystem::exists(configPath)) {
+        return configPath;
+      }
+      configDir = strtok_r(nullptr, ":", &rest);
+    }
+  }
   return std::filesystem::path{SYSTEM_CONFIG_FILE_PATH};
```

Comment:
Since (depending on the prefix used) this will break the spec, I wouldn't mind adding a comment explaining the reasons behind it.